Editor/Menus/CinemachineMenu.cs

```diff
@@ -144,7 +145,7 @@ private static void CreateDollyTrackWithCart()
             if (SceneView.lastActiveSceneView != null)
                 go.transform.position = SceneView.lastActiveSceneView.pivot;
             Undo.RegisterCreatedObjectUndo(go, "create track");
-            CinemachineSmoothPath path = go.GetComponent<CinemachineSmoothPath>();
+            SplineContainer path = go.GetComponent<SplineContainer>();
```

Comment:
This will return null because you haven't added the component.  This needs to be done a few lines above, instead of creating the CinemachineSmoothPath component

Reply:
Did you try running this code by selecting the Create Dolly Cart item in the Cinemachine menu?

Reply:
My bad, should be fixed in next commit.